### ISSUE TITLE
Re-importing exported mesh for ME2 for Jack breaks animations

### ISSUE THREAD

**Issue by Mgamerz** [ME3Tweaks/LegendaryExplorer#168]
Reported by Filipi on Discord

**Describe the bug**

Hi, I've come across a problem I can't seem to solve. Replacing Jack's character mesh in ME2 with any other mesh causes the idle and walking animations to break as in the picture. I've replaced a few other meshes and they all worked fine, but Jack's doesn't. This happens if I export the original ME2 mesh with Meshplorer, re-import into UDK to save as .upk and replace it. What's causing this to happen?

Animations for Jack are broken after re-import in game.
![issue](https://i.imgur.com/6enKaNX.jpg)

**To Reproduce**
Be specific. Reports must include tool names, filenames, export numbers and more so that we can reproduce the issue. **Do not use generic "use any file" or "any tool". Be specific. If specific reproducible steps are not included, we may simply close the issue as not reproducible.**

1. Export Jack's mesh from ME2 in the file BioH_Convict_00.pcc using the included UModel in Meshplorer. Export 4571
2. Import mesh back into UDK
3. Import mesh from UDK into ME3Explorer and save package again
4. In game jack will be messed up

**Expected behavior**
Jack should not be this flexible.

**Version information:**
4.1.1.314

**Other information**
Note that if I were to import any other mesh, for example Samara which has the same skeleton, it would also break.
And if I import the Jack mesh (that I've exported with UDK) into any other .pcc, like Samara, it will work just fine.
It's just Jack's .pcc that seems to have broken animation when replaced with anything else.

**Comment by scottina123** [comment]
![MassEffect3_2020-04-17_16-42-11](https://user-images.githubusercontent.com/54486002/79614156-f37e1980-80cd-11ea-9cd0-f86ea5c3da77.jpg)
Seems like the same issue happend when you try to replace a model using meshplorer in ME3. Tested with 4/17 nightly

**Comment by Mgamerz** [comment]
The latest beta now includes an import LOD feature that hopefully works around this issue. Can you test this out? You don't select a LOD like you used to, it will simply choose the top one and strip the others from the existing mesh (I don't think anyone ever replaced lower LODs).

**Comment by mirh** [comment]
This worked perfectly with miranda (which as I said, didn't break *per se*, but was getting the same results when you tried to swap the outfit)
~~Though, it's a bit lacking that the LOD button is only present in the "tools" dropdown menu, and not also in the right click context menu.~~

**Comment by Mgamerz** [comment]
Has this issue been fixed? It's been some time since I've seen this issue crop up.

**Comment by mirh** [comment]
You still have to select "replace single LOD", rather than "replace mesh". 
Is there no way a single button with some auto-selection logic behind could exist?

**Comment by Mgamerz** [comment]
I assume if we had auto select logic we would know what the issue was and
be able to correct it in the original. It's something to do with us
trusting UDK too much. Single LOD only pulls in the LOD model (there's
other data about the mesh, such as origin, outside of the LOD), where as
mesh pulls in the entire export. But UDK reorders everything so comparing
is really difficult.

Did you verify this recently? The other modders with this issue haven't
shown up in a few months.

**Comment by mirh** [comment]
Yes, I just checked this with a fresh export-edit-import on 1623, blender and UDK 2015.

**Comment by Mgamerz** [comment]
@SirCxyrtyx looked into this and reported that the issue is likely due to the bones being re-ordered. We aren't sure yet if it's due to UModel export or UDK. Not sure how possible this is to fix. Might be we could just apply our single LOD import style to all LODs in the UDK import, I suppose. There's very few times where more than one LOD will be imported (I'm pretty sure most users just import a model that only has one LOD), but it is a good idea to support due to some areas being heavy on resources.
